scripts/ci-ecr-pull

```diff
@@ -15,49 +15,4 @@
 # limitations under the License.
 set -e
```

Comment:
do we need this script at all?

Reply:
we don't, but we currently reference it in internal scripts that will take some time to deploy away, so safest to just leave it here as a dummy for now